nginx-serve/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM node:18-bullseye AS runtime
```

Comment:
We might also need to add documentation for nginx-serve.

@samshara has started documentation on CONTRIBUTING so we might need to keep that up-to-date.

Reply:
This is intended for use for our local cluster only. Also, a bit experimental.
But I don't have a strong opinion on it. Feel free to add.